app.html

```diff
@@ -558,16 +560,37 @@
 
 function SaveFont() {
   var font_select = document.getElementById('font_select');
-  console.log("SAVE FONT " + font_select.value);
-  if (presets[current_preset_num].FONT != font_select.value) {
-    presets[current_preset_num].FONT = font_select.value;
-    Send("set_font " + font_select.value);
+  var addon_select = document.getElementById('font_select2');
+  var font_path = font_select.value;
+  if (addon_select.value != "none"){
+    font_path += ";" + addon_select.value;
+  }
+  console.log("SAVE FONT " + font_path);
+  if (presets[current_preset_num].FONT != font_path) {
+    presets[current_preset_num].FONT = font_path;
+    Send("set_font " + font_path);
   }
 }
 
 function make_font_field(value) {
-  ret="<div class=edit-select><span class=title>font<br><select class=myselect id=font_select name=font_select onchange='SaveFont()'>";
-  ret+= make_select(font_lines, value);
+  var tmp = value.split(";");
+  var font_ = value;
+  var addon_ = "none"
+  if (tmp.length > 1){
+    for (var i = 0; i < font_lines.length; i++) {
```

Comment:
Could you add some comments to explain what these are for.
I think I get it, but it's probably better to explain your intentions than having to infer it from the code.

Reply:
done